src/main/java/org/sunbird/user/service/UserUtilityServiceImpl.java

```diff
@@ -161,8 +162,9 @@ public Map<String, Object> getUsersDataFromUserIds(List<String> userIds, List<St
 
 		try {
 			String url = props.getSbUrl() + props.getUserSearchEndPoint();
-			HttpEntity<?> requestEnty = new HttpEntity<>(requestObj, headers);
-			SearchUserApiResp searchUserResult = restTemplate.postForObject(url, requestEnty, SearchUserApiResp.class);
```

Comment:
can you make sure restTemplate is been replaced with outboundRequestHandlerService
and remove the restTemplate object from this class ?